scripts/ci/jobs/check-konflux-pipelines.sh

```diff
@@ -6,18 +6,20 @@
 set -euo pipefail
 
 ensure_create_snapshot_runs_last() {
-    expected_runafter="$(yq eval '.spec.tasks[] | select(.name != "create-acs-style-snapshot") | .name' .tekton/operator-bundle-pipeline.yaml | sort)"
-    actual_runafter="$(yq eval '.spec.tasks[] | select(.name == "create-acs-style-snapshot") | .runAfter[]' .tekton/operator-bundle-pipeline.yaml)"
+    local pipeline_path=".tekton/operator-bundle-pipeline.yaml"
+    local task_name="create-acs-style-snapshot"
+    expected_runafter="$(yq eval '.spec.tasks[] | select(.name != '\"${task_name}\"') | .name' "${pipeline_path}" | sort)"
+    actual_runafter="$(yq eval '.spec.tasks[] | select(.name == '\"${task_name}\"') | .runAfter[]' "${pipeline_path}")"
 
-    echo "➤ .tekton/operator-bundle-pipeline.yaml // create-acs-style-snapshot: task's runAfter contents shall match the expected ones (left - expected, right - actual)."
+    echo "➤ ${pipeline_path} // checking create-acs-style-snapshot: task's runAfter contents shall match the expected ones (left - expected, right - actual)."
     if ! diff --side-by-side <(echo "${expected_runafter}") <(echo "${actual_runafter}"); then
         echo >&2 -e """
```

Comment:
(I saw this before but did not comment.)
I don't think that `bash` has any special triple-quote thing to enable multiline strings like Python does. Folks on the internet say that
```bash
a="""
something
"""
```
would be just three strings concatenated together. The first and the third ones are empty.

Strings in bash are allowed to be multiline so you can write
```bash
a="
something
"
```
to the same effect.